### ISSUE TITLE
Support to generate empty directory

### ISSUE THREAD

**Issue by LucienLee** [jondot/hygen#207]
I would like to generate empty directory without any files. How can it be worked in current hygen?
One possible workaround is generated a `.gitkeep` as the placeholder.

**Comment by manuman94** [comment]
Yes! It would be nice for creating project templates. For example: "uploads" directory should be empty initially but must be there.

**Comment by jondot** [comment]
You can accomplish this using a shell action `mkdir -p <%= yourparam %>`
